src/angular/styles/_typography.scss

```diff
@@ -95,17 +95,13 @@
   --sbb-color-lemon: #{sbb.$sbbColorLemon};
   --sbb-color-green: #{sbb.$sbbColorGreen};
 
-  // Non-Standard Colors
-  --sbb-color-grey: #{sbb.$sbbColorGrey};
-  --sbb-color-grey-medium: #{sbb.$sbbColorGreyMedium};
-
```

Comment:
This is a breaking change for me which should we better not commit right now. Maybe also leave it here forever.

Reply:
Good point, thanks! I agree with leaving it there forever